src/OptionsResolver.php

```diff
@@ -0,0 +1,401 @@
+     *
+     * @param string $option The name of the option
+     *
+     * @throws UndefinedOptionsException When the option is not defined
+     *
+     * @return string|null The default value of the option if set, null otherwise
+     */
+    public function getDescription($option)
+    {
+        if (!$this->isDefined($option)) {
+            throw new UndefinedOptionsException(sprintf('The option "%s" does not exist.', $option));
+        }
+
+        if (!array_key_exists($option, $this->descriptions)) {
+            return null;
+        }
+
+        return $this->descriptions[$option];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setAllowedValues($option, $allowedValues = null)
+    {
+        parent::setAllowedValues($option, $allowedValues);
+
+        $this->allowedValues[$option] = is_array($allowedValues) ? $allowedValues : array($allowedValues);
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addAllowedValues($option, $allowedValues = null)
+    {
+        parent::addAllowedValues($option, $allowedValues);
+
+        if (!is_array($allowedValues)) {
+            $allowedValues = array($allowedValues);
+        }
+
+        if (!isset($this->allowedValues[$option])) {
+            $this->allowedValues[$option] = $allowedValues;
+        } else {
+            $this->allowedValues[$option] = array_merge($this->allowedValues[$option], $allowedValues);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Returns the allowed values of an option.
+     *
+     * @param string $option
+     *
+     * @return array|null
+     */
+    public function getAllowedValues($option)
+    {
+        if (!$this->isDefined($option)) {
+            throw new UndefinedOptionsException(sprintf('The option "%s" does not exist.', $option));
+        }
+
+        if (!array_key_exists($option, $this->allowedValues)) {
+            return null;
+        }
+
+        return $this->allowedValues[$option];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setAllowedTypes($option, $allowedTypes = null)
+    {
+        parent::setAllowedTypes($option, $allowedTypes);
+
+        $this->allowedTypes[$option] = (array) $allowedTypes;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addAllowedTypes($option, $allowedTypes = null)
+    {
+        parent::addAllowedTypes($option, $allowedTypes);
+
+        if (!isset($this->allowedTypes[$option])) {
+            $this->allowedTypes[$option] = (array) $allowedTypes;
+        } else {
+            $this->allowedTypes[$option] = array_merge($this->allowedTypes[$option], (array) $allowedTypes);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Adds a normalizer for the given option.
+     *
+     * @param string   $option
+     * @param \Closure $normalizer
+     *
+     * @return $this
+     */
+    public function addNormalizer($option, \Closure $normalizer)
```

Comment:
If the only purpose of this method is to use `unbind` helper then drop both of them.
Nice workaround, but we gonna drop 5.3 anyway